src/index.html

```diff
@@ -13,6 +13,55 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <!-- <h1>Stars</h1> -->
+    <main class="main">
+      <div class="stars stars--0">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+
```

Comment:
There should be no blank line between parent and child elements. Remove the empty line for better readability.